docs/xrs.md

```diff
@@ -28,7 +28,12 @@ Prevents the Eclipse OpenJ9&trade; runtime environment from handling any interna
 
 Disabling signal handling in the OpenJ9 VM reduces performance by approximately 2-4%, depending on the application.
 
-:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** Setting this option prevents dumps being generated by the OpenJ9 VM for signals such as `SIGSEGV` and `SIGABRT`, because the VM is no longer intercepting these signals. Do not use the `-Xrs` and [`-XX:+HandleSIGABRT`](xxhandlesigabrt.md) options together. An error is thrown if both options are enabled. To resolve this error, one of the options should be disabled.
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** Setting this option prevents dumps being generated by the OpenJ9 VM for signals such as `SIGSEGV` and `SIGABRT`, because the VM is no longer intercepting these signals. Do not use the `-Xrs` option with the following messages:
```

Comment:
```suggestion
:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** Setting this option prevents dumps being generated by the OpenJ9 VM for signals such as `SIGSEGV` and `SIGABRT`, because the VM is no longer intercepting these signals. Do not use the `-Xrs` option with the following options:
```